Changelog.md

```diff
@@ -1,3 +1,6 @@
+**V1.11.12 - Updates**
+- Fix DEC limit code to return degrees instead of stepper positions. Support separate limits being queried.
```

Comment:
IMO also reword to say "Change" instead of "Fix", since it's an interface change not a bug